### PR TITLE
Update database configuration

### PR DESCRIPTION
**Pull request by rioug**
Update database configuration to allow connection from the new metabase server.

### CODE REVIEW

**Review by dacook — COMMENTED** [review]
Can we just replace `custom_hba_metabase`, or is there a need to have both side by side?

File: inventory/host_vars/openfoodnetwork.org.au/config.yml [hunk 1]
```diff
@@ -26,11 +26,14 @@ postgres_listen_addresses:
...
   - "{{ custom_hba_metabase }}"
+  - "{{ custom_hba_new_metabase }}"
+  - "{{ custom_hba_new_metabase_IPv6 }}"
   - "{{ custom_hba_n8n }}"
   - "{{ custom_hba_new_n8n }}"
   - "{{ custom_hba_new_n8n_IPv6 }}"
   - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
-  - { type: hostssl, database: "{{ db }}", user: zapier, address: '167.99.89.242/32', auth_method: md5 } # for metabase (data.openfoodnetwork.org.au)
+  - { type: hostssl, database: "{{ db }}", user: zapier, address: '167.99.89.242/32', auth_method: md5 } # for metabase (data.openfoodnetwork.org.uk)
+  - { type: hostssl, database: "{{ db }}", user: zapier, address: '89.47.50.240/32', auth_method: md5 } # for metabase (data.openfoodnetwork.org)
```
Comment (rioug) [inline inventory/host_vars/openfoodnetwork.org.au/config.yml:36]: I'll clean that up once I am finished with moving the metabase server

**Comment by rioug** [comment]
> Can we just replace `custom_hba_metabase`, or is there a need to have both side by side?

I'd like to have both so I can make sure everything is fine before pulling the plug on the old server.

File: inventory/host_vars/_example.com/config.yml [hunk 2]
```diff
@@ -28,6 +28,8 @@ custom_hba_entries:
   - "{{ custom_hba_n8n }}"
   - "{{ custom_hba_new_n8n }}"
   - "{{ custom_hba_new_n8n_IPv6 }}"
+  - "{{ custom_hba_new_metabase }}"
+  - "{{custom_hba_new_metabase_IPv6 }}"
```
Comment (mkllnk) [inline inventory/host_vars/_example.com/config.yml:32]:
```suggestion
  - "{{ custom_hba_new_metabase_IPv6 }}"
```

**Review by mkllnk — APPROVED** [review]
Great!

**Pull request merged**